airflow/providers/amazon/aws/hooks/emr.py

```diff
@@ -202,6 +202,74 @@ def get_ui_field_behaviour() -> dict[str, Any]:
             },
         }
 
+    def is_cluster_available(self, emr_cluster_id, cluster_states):
```

Comment:
All of these methods are missing doc strings and this one is missing typing.

I also don't see this method used anywhere?

Reply:
Yes, its stale so this method is removed now